tests/scripts/populate_indices_for_cloud_tests.py

```diff
@@ -0,0 +1,142 @@
+import json
+import os
+import time
+import zlib
+
+import marqo
+
+
+def create_settings_hash(settings_dict, **kwargs):
+    """
+    Creates a hash from the settings dictionary and kwargs. Used to ensure that each index is created unique.
+    Size is restricted on 10 characters to prevent having to big index name which could cause issues.
+    """
+    dict_to_hash = settings_dict["index_defaults"] if settings_dict else kwargs
+    combined_str = json.dumps(dict_to_hash, sort_keys=True)
+    crc32_hash = zlib.crc32(combined_str.encode())
+    short_hash = hex(crc32_hash & 0xffffffff)[2:][
+                 :10]  # Take the first 10 characters of the hexadecimal representation
+    print(f"Created index with settings hash: {short_hash} for settings: {dict_to_hash}")
+    return short_hash
+
+
+def populate_indices():
```

Comment:
shouldn't the mapping and, related functionality, of 
```
index_name <> settings <> cloud_testing_index_name 
```
Be available and easy to use for the entire test dir (rather than separating this logic into this file)?